demos/Clamp/main.vala

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env -S vala workbench.vala --pkg libadwaita-1
+
+public void main () {
+    var button_increase = (Gtk.Button) workbench.builder.get_object ("button_increase");
+    var button_decrease = (Gtk.Button) workbench.builder.get_object ("button_decrease");
+    var clamp = (Adw.Clamp) workbench.builder.get_object ("clamp");
+
+    button_decrease.clicked.connect (() => decrease (button_decrease, clamp));
+    button_increase.clicked.connect (() => increase (button_increase, clamp));
+}
+
+public static void increase (Gtk.Button button, Adw.Clamp clamp) {
+    var current_size = clamp.get_maximum_size ();
```

Comment:
Use `int` instead of `var`, as the return type is not obvious.
Prefer properties instead of getters, i.e, `int current_size = clamp.maximum_size`